### ISSUE TITLE
Razer Naga X support

### ISSUE THREAD

**Issue by nicolas3366** [1kc/razer-macos#367]
Hi!
I recently bought a Naga X which currently does not seem to be supported.
How likely is it that Naga X support will be added in the future?

greetings
nicolas

**Comment by dhobi** [comment]
Current state in the openrazer project (which this project ports the code over): https://github.com/openrazer/openrazer/issues/1463

**Comment by maxgmer** [comment]
@dhobi they added support to Razer Naga X, can you check?

**Comment by dhobi** [comment]
@maxgmer Yes, it seems that openrazer has support for Naga X now. All you have to do now is to find someone who:
- has a Naga X
- has a Mac
- is willing to put some effort into porting and testing the code into a PR
- can review and merge this PR
- release a new Razer macOS version

Good luck! 🙂

**Comment by mavonarx** [comment]
[changes_for_naga_x.txt](https://github.com/1kc/razer-macos/files/14808574/changes_for_naga_x.txt)

Make these changes in the librazermacos directory, and rebuild the dmg. 
To build the dmg i needed to change these things: 
[build_changes.txt](https://github.com/1kc/razer-macos/files/14808601/build_changes.txt)
Ah and also make a json file for the device. in devices folder. 
Hope it works for you as well

**Comment by reoccurcat** [comment]
@mavonarx does this work with colors of the mouse etc?? Definitely will try when I get the chance

**Comment by mavonarx** [comment]
Yes it seems to work :)

**Comment by mavonarx** [comment]
https://www.swisstransfer.com/d/3525d8f2-1206-4b43-9452-911ad6708836

Here is the download link for the dmg file if you don't want to build it yourself

**Comment by reoccurcat** [comment]
Lemme give that a try, thank you smsm!!

EDIT: it works, you sir are a genius. Thank you thank you thank you!

**Comment by reoccurcat** [comment]
Also, might I suggest that you create a fork of this repo with your changes? I'm sure it would be greatly appreciated by a lot of people (since I've seen this issue pop up a lot of times in this repository)

**Comment by mavonarx** [comment]
actually good idea, this is my fork https://github.com/mavonarx/razer-macos
